src/cli.rs

```diff
@@ -80,13 +80,16 @@ pub struct Cli {
     #[clap(long, value_name = "MIME_TYPE")]
     pub response_mime: Option<String>,
 
-    /// String specifying what the output should contain.
+    /// String specifying what the output should contain
     ///
-    /// Use `H` and `B` for request header and body respectively,
-    /// and `h` and `b` for response hader and body.
+    ///     'H' request headers
+    ///     'B' request body
+    ///     'h' response headers
+    ///     'b' response body
+    ///     'm' response metadata
```

Comment:
This change doesn't seem to play well with manpages :(

Reply:
We can ignore this since manpages will be overhauled in https://github.com/ducaale/xh/pull/250